src/html/html-script-tag.ts

```diff
@@ -49,6 +50,12 @@ export class ScannedScriptTagImport extends ScannedImport {
           this.warnings);
     } else {
       // not found or syntax error
+      document.warnings.push({
+        code: 'could-not-load',
+        message: `Unable to load import: ${this.error!.message || this.error}`,
```

Comment:
I was under the impression when the scanned document was not found, it failed to analyze. Nonetheless will add an if check for it 😄